app/src/composables/use-m2m.ts

```diff
@@ -42,9 +42,7 @@ export default function useRelation(collection: Ref<string>, field: Ref<string>)
 	const relation = computed(() => {
 		return relations.value.find(
 			(relation) =>
-				relation.collection === junction.value.collection &&
-				relation.field !== junction.value.field &&
```

Comment:
Not sure what is this condition is doing, but it throws an error if reusing same junction collection.
It seems to work fine without it. Tested on M2M list, translations and files.